com.unity.uiwidgets/Runtime/widgets/basic.cs

```diff
@@ -1901,8 +1901,8 @@ public class RawImage : LeafRenderObjectWidget {
                 alignment: alignment,
                 repeat: repeat,
                 centerSlice: centerSlice,
-                invertColors: invertColors,
-                filterMode: filterMode
+                invertColors: invertColors
+                // filterMode: filterMode
```

Comment:
we should add the filterQuality parameter here， making it exactly the same with the codes in flutter?

Reply:
Updated